stable_baselines/sac/sac.py

```diff
@@ -314,6 +314,7 @@ def setup_model(self):
                 self.summary = tf.summary.merge_all()
 
     def _train_step(self, step, writer, learning_rate):
+        del step
```

Comment:
if step is not used anymore, why not removing it from the arguments?

Reply:
good point! I will do that instead, wasn't sure if it followed a train_step template used by other algorithms as well, but it appears that is not the case

Reply:
removed step, and fixed issue of wrapping replay_buffer on each call to learn (now only happens on the first one). Although let me know if you think it is better to check if self.model.replay_buffer is wrapped instead using `isinstance(self.model.replay_buffer, HindsightExperienceReplayWrapper)`